nrfu_tests/test_definition.yaml

```
@@ -0,0 +1,12 @@
+- name: nrfu_tests
+  testcases:
+  - name: test_port_channel_interface_member_details
```

```suggestion
  - name: test_port_channel_member_interface_details
```

Also please check formatting of parameter in test definitions.